tests/testthat/test-get_pvol_us.R

```diff
@@ -32,33 +32,25 @@ test_that("NEXRAD polar volume correct time is downloaded", {
   ))
 })
 
-test_that("Mixed radar vector (single timestamp)", {
-  skip_if_offline()
-  suppressMessages(pvols <- getRad::get_pvol(c("KABR", "czska"), time_utc))
-  expect_true(is.list(pvols))
-  expect_gt(length(pvols), 0)
-  expect_true(all(purrr::map_lgl(pvols, ~ inherits(.x, "pvol"))))
-})
+
 test_that("Correct error is given when no near data is found", {
   expect_error(get_pvol("KABX", as.POSIXct("1970-1-1")), class = "getRad_error_us_no_scan_found")
 })
 
-test_that("Mixed radar vector + 9 minute interval", {
-  skip_if_offline()
-  suppressMessages(pvols <- getRad::get_pvol(c("KABR", "czska"), dt_int))
-  expect_true(is.list(pvols))
-  expect_gt(length(pvols), 2)
-  expect_true(all(purrr::map_lgl(pvols, ~ inherits(.x, "pvol"))))
-})
+
 test_that("Caching of keys works", {
```

Comment:
nitpick: I'm not the biggest fan of the `system.time()` based testing, there might be slowdowns on the test runner. I see why you skip this test on CRAN. 

Is checking for the presence for the key in the cache not enough? 

We could also use withr to set a local option to overwrite the default cache, and that way we'll be super certain both of cache location and that it starts as empty.

Reply:
I think test that is anyway run in a separate environment. I already check the files are not present now (see `expect_false`). I do the time based check to ensure the cache is actually used. I think this design should be pretty robust looking for a relative speed increase